### DISCUSSION TITLE
Countdown Component is not working

### DISCUSSION THREAD

**Discussion by Aiman-Shafi** [saadeghi/daisyui#2757] [Q&A]
I'm trying to implement the countdown by using the provided JSX but doesn't seem to be working it's static. Let me know if i'm missing anything out on docx **https://daisyui.com/components/countdown/**

**Comment by saadeghi** [comment]
...   
you're missing the first and only paragraph of docs on the page you linked:

>Countdown gives you a transition effect of changing numbers.  
You need to change to `--value` CSS variable using JS. Value must be a number between 0 and 99.
